http-clients/src/test/resources/service-config-example.yml

```diff
@@ -0,0 +1,73 @@
+auth:
+  service: # just a standard ServiceConfig block
+    security:
+      trustStorePath: src/test/resources/trustStore.jks
+    uris:
+      - https://some.internal.url:8643/service/api
+  oauth2:
```

Comment:
`getOAuth2()` isn't called anywhere, so this is redundant. the whole `auth` block is dangling mid-air without purpose.